src/app/clusters/scenes-server/SceneHandlerImpl.cpp

```diff
@@ -130,13 +134,20 @@ void CapAttributeID(typename app::NumericAttributeTraits<Type>::WorkingType & Va
         maxValue                                        = ConvertDefaultValueToWorkingValue<Type>(minMaxValue->maxValue);
     }
 
-    if (minValue > Value)
+    if (metadata->IsNullable() && (minValue > value || maxValue < value))
+    {
+        // If the attribute is nullable, the value can be set to NULL
```

Comment:
But if the value is nullable your min/max above are wrong...

Reply:
True but since without a specific metadata min/max, the max for a uint and min for an int will be UINTMAX - 1 and INTMIN + 1, doesn't that end up being the same?
Say for uint8, with a max of 0xFE, the only value above max will already be considered NULL so no changes will be required.
Same for int8 where -128 would be considered a NULL value without needing to enter this if condition.

Reply:
Yes, you are right.  Maybe just add a comment for now that if the value is outside the nullable range but inside the non-nullable range then it's already set to null so no need to do anything?